tests/integration/test_vid_club.py

```diff
@@ -23,23 +29,20 @@ def json():
 
 
 @pytest.mark.init
-def test_create_club_class():
-    vc = VidClub()
-    assert vc
+def test_default_credential_param():
+    assert vc.credentials != None and type(vc.credentials) == dict
```

Comment:
Please add docstring to test function that will shortly summarize what they are testing